### DISCUSSION TITLE
Invalid client hash errors with the new 2022.226.0 update

### DISCUSSION THREAD

**Discussion by Walavouchey** [ppy/osu#16994] [Q&A] [answered]
Me and the other people I've tried the new update with are getting client hash errors when trying to play multiplayer. Some people are getting it to work, and spectating them works fine (although it doesn't seem to proceed to the results screen).

I've attached logs of a session where I create a room, start the match and get denied because of the error (multiplayer logs)

Submitting a score in solo also doesn't seem to work, since nothing appears on the "recent" tab of my [profile page](https://osu.ppy.sh/users/5773079) (solo logs)

The logs are after deleting `$APPDATA$/local/osulazer` and reinstalling via the installer on the GitHub release page in a failed effort to fix some kind of local issue.

[logs multiplayer.zip](https://github.com/ppy/osu/files/8146580/logs.multiplayer.zip)
[logs solo.zip](https://github.com/ppy/osu/files/8146579/logs.solo.zip)

On looking at the logs myself, I don't see many clues for why it might've happened, but I hope it's reproducible since many people are getting the same issue

Video of the issue happening:

https://user-images.githubusercontent.com/36758269/155843407-4850c52b-7ad1-43bd-b42a-c41fb425a619.mp4

I was suspecting maybe the update was pulled or maybe the servers needed a bit to whitelist the new hash, but no such thing was announced. After waiting a bit and still seeing this error (along with other people currently in multiplayer) I'm making this report now

**Comment by bdach** [comment]
Thanks, have raised internally.

**Comment by peppy** [comment, marked as answer]
Which OS is this?

**Reply by bdach** [reply]
Windows (both in logs in OP and in my testing)

**Reply by peppy** [reply]
Can you confirm linux is working correctly?

Looks like somehow an old binary may have been pushed in the build... maybe github doesn't sort releases correctly..

**Reply by bdach** [reply]
Need to reboot to check, give me a minute...

**Reply by peppy** [reply]
I've fixed this for the windows builds. Unsure if it affects macOS as well, will check now.

**Reply by bdach** [reply]
Very late due to *reasons* but linux is also fine (as per discord conversation). Should be all OK now.
